README.md

```diff
@@ -117,14 +115,18 @@ rest:
   listen: "127.0.0.1:8443"
   pkcs12: "example.p12"
   prefix: "api"
+peer_2_peer:
+  trusted_peers: []
+  public_access: "/ip4/127.0.0.1/tcp/8080"
```

Comment:
Would it be better to use a property map with sensible defaults for some of the fields (IPv4/TCP) instead of this microformat?